homeassistant/components/xiaomi_miio/air_quality.py

```diff
@@ -33,6 +35,8 @@
 PROP_TO_ATTR = {
     "carbon_dioxide_equivalent": ATTR_CO2E,
     "total_volatile_organic_compounds": ATTR_TVOC,
+    "temperature": ATTR_TEMP,
```

Comment:
These should be individual sensor entities and not attributes of the air quality sensor.

Reply:
I understand where you are coming from, however `carbon_dioxide` or `particulate_matter_2_5` are also just attributes of the air quality sensor. Should they be individual sensor entities as well?

Reply:
No, those are part of the air quality entity model.

https://developers.home-assistant.io/docs/en/entity_air_quality.html

Humidity and temperature are device classes of the sensor entity model.

https://developers.home-assistant.io/docs/en/entity_sensor.html#available-device-classes

We should follow our architecture models as far as possible.

Reply:
Yeah you are completely right.